server/js/topics.js

```diff
@@ -0,0 +1,77 @@
+class DAS {
+  static data = 'data';
+  static start = 'start';
+  static stop = 'stop';
+}
+
+class BOOST {
+  // Start/stop
+  static start = 'boost/start';
+  static stop = 'boost/stop';
+
+  // BOOST output
+  static recommended_sp = 'boost/recommended_sp';
+  static predicted_max_speed = 'boost/predicted_max_speed';
+
+  // Plan generation
+  static generate = 'boost/generate';
+  static generate_complete = 'boost/generate_complete';
+
+  // Distance calibration
+  static calibrate = 'boost/calibrate';
+  static calibrate_reset = 'boost/calibrate/reset';
+}
+
+class Camera {
+  // Getting/setting overlays
+  static get_overlays = 'camera/get_overlays';
+  static set_overlay = 'camera/set_overlay';
+  static push_overlays = 'camera/push_overlays';
+
+  // Error topic
+  static errors = 'camera/errors';
+
+  // Messages to be shown on DAShboard
+  static overlay_message = 'camera/message';
+
+  // Recording
+  static recording = 'camera/recording/+';
+  static recording_start = 'camera/recording/start';
+  static recording_stop = 'camera/recording/stop';
+
+  // Camera status
+  static status_recording = 'status/camera/recording';
+  static status_video_feed = 'status/camera/video_feed';
+  static status_camera = 'status/camera';
+}
+
+class WirelessModule {
+  static base = '/v3/wireless_module/';
+
+  battery(module_id = '+') {
```

Comment:
Since the `WirelessModule` topics depend on a `module_id` property, perhaps it would make more sense to add it as an instance property and instantiate `WirelessModule` when accessing the topics.
Alternatively, we could use `WirelessModule` as a topic factory which has a method `id()` that returns an object for the topics of that module id (e.g. `WirelessModule.id(1).data`).

Reply:
I see what you're saying, but I also feel it would be weird to have to go `new WirelessModule` every time we want to refer to a constant topic. It would make the usage inconsistent with the other topics and with the `common` repo `topics.py`.

Reply:
Agreed. What about the alternative I mentioned with `WirelessModule.id(1).data`?

Reply:
What's currently in `topics.js` is as far as I can see exactly the same as `topics.py`. I'm happy to go either way (tbh I don't see why one is better than the other) but I think if we change it here it should also be changed in monash-human-power/common/pull/12 and then all further usages in other repos. Reckon it's worth changing?

Reply:
Oh I didn't realise this was changed in the new PR. In that case, it is not at all consistent with what we had before because it introduces methods for the class when all the other classes use literals.
Noted in [common #12](https://github.com/monash-human-power/common/pull/12#discussion_r539241066)